submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt

```diff
@@ -24,8 +24,8 @@ class SubmissionService(
     private val extSubmissionSubmitter: ExtSubmissionSubmitter,
     private val submissionSubmitter: SubmissionSubmitter,
     private val eventsPublisherService: EventsPublisherService,
-    private val fileStorageService: FileStorageService,
     private val submissionPersistenceService: SubmissionPersistenceService,
+    private val submissionRequestFinalizer: SubmissionRequestFinalizer,
```

Comment:
I don't like very much the idea of including the `SubmissionRequestFinalizer` here since it's part of the submission workflow